frontend/components/board.js

```diff
@@ -0,0 +1,167 @@
+var React = require('react')
+
+function Square(props) {
+  return (
+    <button className="square" id={props.value}>
+    </button>
+  );
+}
+
+module.exports = React.createClass({
+  getInitialState: function() {
+    var squares = Array(100).fill(null);
+    var isBlack = true;
+    for(var i=0; i<squares.length; i++) {
+      if(i%10 == 0) {
+        isBlack = !isBlack
```

Comment:
Up to you. But why not consolidate this logic of setting the bool then checking the bool to just setting the array[i] to the proper color string?